### PR TITLE
raidboss japanese localize

### PR DESCRIPTION
**Pull request by r4ha**
btw, is there any way to host overlay url with github pages? I wanna share my rep to friends instantly. but it doesn't work...

### CODE REVIEW

**Comment by quisquous** [comment]
This looks great, can you fix the two lint errors from travis ci? https://travis-ci.com/quisquous/cactbot/jobs/221345257

`npm run lintfix` should fix them automatically if you have them installed

I am using github pages to share cactbot master branch, e.g. having people use https://quisquous.github.io/cactbot/ui/raidboss/raidboss.html as their raidboss url.  You may be able to do the same with your fork's master branch?

**Comment by r4ha** [comment]
oops, fixed it.

thanks ill try it!

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,5 @@ package-lock.json
 .idea/*
 .idea/
 *.iml
-.DS_Store
\ No newline at end of file
+.DS_Store
+user/raidboss-example.js
```
Comment (quisquous) [inline .gitignore]: Can you remove this from here? The example is part of the repository, so I don't want to ignore changes to it.

File: ui/raidboss/data/triggers/e3s.js [hunk 2]
```diff
@@ -230,93 +268,150 @@
...
       delaySeconds: 25,
       infoText: {
         en: 'Move In, Avoid Defamation',
+        ja: '前にノックバック',
       },
     },
     {
       id: 'E3S Sweeping Waters',
-      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Sundering Waters from (?:.*) for (.*) Seconds/,
+      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of 断絶の兆し from (?:.*) for (.*) Seconds/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e3s.js]: Can you change this to regexJa and not replace the original?

File: ui/raidboss/data/triggers/e3s.js [hunk 3]
```diff
@@ -230,93 +268,150 @@
...
       },
       infoText: {
         en: 'Tank Cone',
+        ja: '断絶',
       },
     },
     {
       id: 'E3S Sweeping Waters',
-      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Sundering Waters from (?:.*) for (.*) Seconds/,
+      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of 断絶の兆し from (?:.*) for (.*) Seconds/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e3s.js]: Can you change this to regexJa and not replace the original?

File: user/raidboss-example.js [hunk 4]
```diff
@@ -61,16 +61,16 @@ Options.AlarmSound = '../../resources/sounds/BigWigs/Alarm.ogg';
 Options.LongSound = '../../resources/sounds/BigWigs/Long.ogg';
 
 // Volume between 0 and 1 to play the InfoSound at.
-Options.InfoSoundVolume = 1;
+Options.InfoSoundVolume = 0.05;
```
Comment (quisquous) [inline user/raidboss-example.js]: Can you undo the changes to this file?

File: ui/raidboss/data/triggers/e2s.js [hunk 5]
```diff
@@ -45,87 +47,105 @@
       id: 'E2S Entropy',
       regex: / 14:3E6F:Voidwalker starts using (?:Entropy|)/,
       regexFr: / 14:3E6F:Marcheuse Du Néant starts using (?:Entropie|)/,
+      regex: / 14:3E6F:ヴォイドウォーカー starts using (?:エントロピー|)/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e2s.js]: this should be regexJa, I think.

File: ui/raidboss/data/triggers/e3s.js [hunk 6]
```diff
@@ -21,52 +22,61 @@
...
       infoText: {
         en: 'aoe',
         fr: 'Dégâts de zone',
+        ja: 'AoE',
       },
     },
     {
       id: 'E3S Tidal Rage',
       regex: / 14:3FDE:Leviathan starts using Tidal Rage/,
+      regex: / 14:3FDE:リヴァイアサン starts using タイダルレイジ/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e3s.js]: this should be regexJa

File: ui/raidboss/data/triggers/e3s.js [hunk 7]
```diff
@@ -101,18 +114,22 @@
       id: 'E3S Undersea Quake Outside',
       regex: / 14:3FEF:Leviathan starts using Undersea Quake/,
       regexFr: / 14:3FEF:Léviathan starts using Séisme Sous-Marin/,
+      regex: / 14:3FEF:リヴァイアサン starts using アンダーシークエイク/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e3s.js]: this should be regexJa

File: ui/raidboss/data/triggers/e3s.js [hunk 8]
```diff
@@ -123,65 +140,80 @@
...
+      regexJa: / 14:3FE2:リヴァイアサン starts using 猛烈なる波動/,
       infoText: {
         en: 'Stack, Bait Puddles',
+        ja: '集合',
       },
     },
     {
       id: 'E3S Drenching Pulse Puddles',
       regex: / 14:3FE2:Leviathan starts using Drenching Pulse/,
+      regex: / 14:3FE2:リヴァイアサン starts using 猛烈なる波動/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e3s.js]: this should be regexJa

File: ui/raidboss/data/triggers/o11s.js [hunk 9]
```diff
@@ -402,8 +403,8 @@
         'Wave Cannon Kyrios': '波動砲P',
 
         // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o11s.js]: Can you remove this FIXME comment while you're here?

File: ui/raidboss/data/triggers/o8s.js [hunk 10]
```diff
@@ -275,14 +299,15 @@
...
+        ja: '片翼',
       },
     },
     {
       id: 'O8S Ultimate Embrace',
       regex: / 14:2910:Kefka starts using Ultimate Embrace on (\y{Name})/,
       regexDe: / 14:2910:Kefka starts using Ultima-Umarmung on (\y{Name})/,
       regexFr: / 14:2910:Kefka starts using Étreinte Fatidique on (\y{Name})/,
-      regexJa: / 14:2910:ケフカ starts using アルテマte Embrace on (\y{Name})/,
+      regexJa: / 14:2910:ケフカ starts using 終末の双腕 on (\y{Name})/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o8s.js]: Can you change the regex below to be regexKo?

**Comment by r4ha** [comment]
Sorry for bothering you. maybe it should be ok...

**Comment by quisquous** [comment]
> Sorry for bothering you. maybe it should be ok...

It's no problem at all! Thank you for all the changes.  I'm happy to merge any more translations you would like to do.
